modules/nf-core/deepvariant/main.nf

```diff
@@ -12,6 +12,7 @@ process DEEPVARIANT {
     tuple val(meta2), path(fasta)
     tuple val(meta3), path(fai)
     tuple val(meta4), path(gzi)
+    tuple val(meta5), path(par_bed)
```

Comment:
can you update the meta.yml with the new param

Reply:
Oh, good spot

Reply:
Thanks, fixed!